Gems/Atom/Feature/Common/Code/Source/Mesh/MeshFeatureProcessor.cpp

```diff
@@ -240,6 +240,13 @@ namespace AZ
 
             return {};
         }
+        
+        const MeshDrawPacketLods& MeshFeatureProcessor::GetDrawPackets(const MeshHandle& meshHandle) const
+        {
+            static MeshDrawPacketLods empty;
```

Comment:
There is at least one vector in there, but the MeshDrawPacket just uses default constructors, so there should be no use of allocators or something like that. I can make it a member though, just to future-proof it in case something dangerous is added someday.

Reply:
I changed it to be a member.